PyInstaller/compat.py

```diff
@@ -155,7 +155,10 @@
 # List of suffixes for Python C extension modules.
 if is_py2:
     # TODO implement getting extension suffixes for Python 2 or older Python 3.
-    EXTENSION_SUFFIXES = ['.so']
+    if is_win:
+        EXTENSION_SUFFIXES = ['.pyd', '.dll']
```

Comment:
Hmm. I was under the impression that .dll files could be imported, but that's not actually the case. It should be just `['.pyd']`.